microceph/database/config.mapper.go

```diff
@@ -110,7 +110,7 @@ func GetConfigItems(ctx context.Context, tx *sql.Tx, filters ...ConfigItemFilter
 	var err error
 
 	// Result slice.
-	objects := make([]ConfigItem, 0)
+	var objects []ConfigItem
```

Comment:
The *.mapper.go files are generated by lxd-generate, we should avoid editing them if at all possible. I would suggest to report this issue upstream with the lxd project

Reply:
Hmm, that's fair... reversing the changes.